release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -677,6 +710,12 @@ If you have a log consumer that is expecting the capnslog format, you might need
 {"level":"warn","ts":"2021-06-14T13:13:23.243Z","caller":"etcdserver/util.go:163","msg":"apply request took too long","took":"163.262994ms","expected-duration":"100ms","prefix":"read-only range ","request":"key:\"/kubernetes.io/namespaces/default\" serializable:true keys_only:true ","response":"range_response_count:1 size:53"}
 ----
 
+[discrete]
+[id="ocp-4-8-daemonset-merge-lso"]
+==== Multiple daemonsets merged for LSO
+
+In {product-title} 4.8, multiple daemonsets have been merged for Local Storage Object (LSO), so that when you create a local volume custom resource, only `daemonset.apps/diskmaker-manager` is created.
+
 [id="ocp-4-8-deprecated-removed-features"]
```

Comment:
I don't think this needs to be called out in release notes. As such it is not a user facing change.

Reply:
Jan and Duncan decided in favor of adding it.We can discuss this again in the Monday standup, but I'll need a definitive answer soon since doc freeze is Jun 25th.

Reply:
Hemant has agreed that documenting this issue in the "Notable Technical Changes" section is acceptable.